ztp/tools/policy-object-template-diff/Makefile

```diff
@@ -0,0 +1,15 @@
+GOCMD=go
+BINARY_NAME=policy-object-template-diff
+ACM_PATH?=""
+ZTP_PATH?=""
+
+build:
+	mkdir -p out/bin
+	GO111MODULE=on $(GOCMD) build -mod vendor -o out/bin/$(BINARY_NAME) .
```

Comment:
Do we need to use `-o` or can we rely on $GOBIN being set right?  I think $BINARY_NAME will default to the name of the directory that main.go is in, so it should already be called `policy-object-template-diff`?

Reply:
` -o` just to make sure we know where the binary lands so that I can be explicit with the `run` and `clean` command below (not relying CI $GOPATH/bin or $GOBIN or $HOME/go/bin)

Made the name (policy-object-template-diff) explicit  at the top just to help with the `run` command.